physionet-django/events/forms.py

```diff
@@ -1,7 +1,7 @@
 from django import forms
 
 from events.widgets import DatePickerInput
-from events.models import Event
+from events.models import Event, EventApplications
```

Comment:
Sorry, should have picked this up before, but for consistency with other models, should `EventApplications` be `EventApplication`?

Reply:
Yes, i agree. My bad. For some weird reason, Plural made sense to me. Pushing changes right away